examples/web/menu.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
```

Comment:
I think we can add the same header used in the PyGstc Playback Example.

Reply:
I agree